src/core/imageUtils.js

```diff
@@ -17,3 +17,11 @@ export const getPreviewImage = (addon, { index = 0, full = true } = {}) => {
   }
   return null;
 };
+
+export const getPreviewIndexBySize = (addon, size = 720) => {
```

Comment:
Since this module isn't using Flow, you need to be careful about optional parameters. I would rather see it done like this:

```js
export const getPreviewIndexBySize = (addon, { size = 720 } = {}) => {
}
```